src/walk-manifest.js

```diff
@@ -147,125 +168,160 @@ const walkPlaylist = function(options) {
       visitedUrls = [],
       requestTimeout = 1500,
       requestRetryMaxAttempts = 5,
+      dashPlaylist = null,
```

Comment:
In `mpd-parser` we really only mock child playlists. So if we see `dashPlaylist` here we know:
1. That this playlist won't have a uri, or anything to download
2. That this playlist has already been parsed